ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -1281,6 +1281,12 @@ struct TSchemeQuotas : public TVector<TSchemeQuota> {
     mutable size_t LastKnownSize = 0;
 };
 
+enum class EUserFacingStorageType {
```

Comment:
This enum highlights that we categorize numerous storage pool kinds into just two reported storage types: HDD and SSD.